src/main/java/com/keunsori/keunsoriserver/domain/auth/controller/AuthController.java

```diff
@@ -19,9 +21,8 @@ public class AuthController {
     private final AuthService authService;
 
     @PostMapping("/login")
-    public ResponseEntity<Void> login(@Valid @RequestBody LoginRequest loginRequest, HttpServletResponse response){
-        authService.login(loginRequest.studentId(), response);
-        return ResponseEntity.ok().build();
+    public ResponseEntity<LoginResponse> login(@Valid @RequestBody LoginRequest loginRequest, HttpServletResponse response){
+        return ResponseEntity.ok(authService.login(loginRequest.studentId(), response));
     }
```

Comment:
```suggestion
    public ResponseEntity<LoginResponse> login(@Valid @RequestBody LoginRequest loginRequest, HttpServletResponse response){
        LoginResponse loginResponse = authService.login(loginRequest.studentId(), response);
        return ResponseEntity.ok(loginResponse);
    }
```

가독성을 위해서 서비스 메서드의 리턴값을 변수로 빼내주면 좋을 것 같아요. 다른 컨트롤러 코드가 어떻게 짜여있는지 참고해주셔도 좋을 것 같습니다!
사소하긴 하지만, 전체적인 코드 스타일이 통일되어 있으면 나중에 내가 읽거나, 추후에 프로젝트에 새로 들어오는 사람이 읽을 때 더 편해서 스타일을 하나로 맞춰주는 편이 좋아요 ㅎㅎ